packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/cards/WelcomeCard.tsx

```diff
@@ -1,29 +1,90 @@
 import React, { FC } from 'react';
 import { Trans } from 'react-i18next';
 import automationIcon from 'src/modules/Overview/images/automation.svg';
+import { DetailsItem } from 'src/modules/Providers/utils';
 import { useForkliftTranslation } from 'src/utils/i18n';
 
 import { V1beta1ForkliftController } from '@kubev2v/types';
-import { Card, CardBody, Split, SplitItem, Text, TextVariants } from '@patternfly/react-core';
+import {
+  Card,
+  CardActions,
+  CardBody,
+  CardHeader,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  Split,
+  SplitItem,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
 
 type OverviewCardProps = {
   obj?: V1beta1ForkliftController;
   loaded?: boolean;
   loadError?: unknown;
+  onHide?: () => void;
 };
 
-export const OverviewCard: FC<OverviewCardProps> = () => {
+export const OverviewCard: FC<OverviewCardProps> = ({ onHide }) => {
   const { t } = useForkliftTranslation();
+  const actionDropdownItem: any[] = [];
 
+  const [menuIsOpen, setMenuIsOpen] = React.useState(false);
+  const onToggle = () => setMenuIsOpen((open) => !open);
+
+  if (onHide) {
```

Comment:
on hide is a method, you use it here as a boolean, this it not a nice use of a method

Reply:
This is for checking that the method is not null since it's optional. If not set then there is no point on displaying this kebab menu option...

Reply:
yes, but `onHide` is a method, you use the fact it can be `undefined`  or function,
```suggestion
  const hasHideAction = !!ohHide;
  
  if (hasHideAction) {
```

i like this verbose way better because this way the `hasHideAction` const is boolean, and the if check true/false instead of undefined/function